facebook-commerce.php

```diff
@@ -315,7 +315,7 @@ public function __construct( WC_Facebookcommerce $facebook_for_woocommerce ) {
 
 				add_action( 'add_meta_boxes', 'WooCommerce\Facebook\Admin\Product_Sync_Meta_Box::register', 10, 1 );
 
-				add_action( 'add_meta_boxes', [ $this, 'display_batch_api_completed' ], 10, 2 );
+				add_action( 'add_meta_boxes_product', [ $this, 'display_batch_api_completed' ], 10, 2 );
```

Comment:
We should use `add_meta_boxes_{$post_type}` hook instead of `add_meta_boxes` . The order page triggers `add_meta_boxes` with `screen_id` and `order` instead of `$post_type` and `$post`.